clang/unittests/Parse/ParseHLSLRootSignatureTest.cpp

```diff
@@ -252,6 +252,32 @@ TEST_F(ParseHLSLRootSignatureTest, ValidSamplerFlagsTest) {
   ASSERT_TRUE(Consumer->isSatisfied());
 }
 
+TEST_F(ParseHLSLRootSignatureTest, ValidParseRootConsantsTest) {
```

Comment:
```suggestion
TEST_F(ParseHLSLRootSignatureTest, ValidParseRootConstantsTest) {
```

nit: Also, shouldn't this be, `ValidParseEmptyRootConstantsTest`, then we add more tests later on ?

Reply:
Once we add the mandatory parameters then having an empty `RootConstants` is no longer valid, so the idea is that we can merge them into one